testutil/fixture.go

```diff
@@ -40,6 +43,15 @@ type Fixture interface {
 	// random or correspond to nodes in a test network which have keyrings.
 	Signers() []sdk.AccAddress
 
+	// InitGenesisHandler is a function to get a module InitGenesisHandler.
+	InitGenesisHandler(moduleName string) module.InitGenesisHandler
+
+	// ExportGenesisHandler is a function to get a module ExportGenesisHandler.
+	ExportGenesisHandler(moduleName string) module.ExportGenesisHandler
+
+	// Codec is the app ProtoCodec.
+	Codec() *codec.ProtoCodec
```

Comment:
Why do we need all this here?

Reply:
To use them in tests, see in `x/group/server/testsuite/` but happy to discuss any other solutions that would look better to you.

Reply:
How about we simply have `InitGenesis()` and `ExportGenesis()` methods? Rather than being able to look up the handler by module. Also consider we want to use these same `Fixture`s for `Network` based tests in the future (#246).

Reply:
The thing is that we can register multiple modules as part of `FixtureFactory` `Setup()` which is why I added them as look up methods.

Reply:
I don't think that approach will work using a `Network` fixture. Would `InitGenesis` and `ExportGenesis` also work for the tests?

Reply:
When you say `InitGenesis()` and `ExportGenesis()` methods, could you elaborate on the params and return values? 
Do you see them as being similar to the manager  `InitGenesis()` and `ExportGenesis()`, ie they would actually execute  init or export genesis (instead of simply returning a function)?

Reply:
I've updated it so we have `InitGenesis` and `ExportGenesis` methods instead.